src/setupProxy.js

```diff
@@ -4,7 +4,7 @@ module.exports = function (app) {
     app.use(
```

Comment:
are we committing set up proxy as well?

Reply:
It is for this rel branch won't be committing it into main